optimum/gptq/quantizer.py

```diff
@@ -123,6 +124,8 @@ def __init__(
             cache_block_outputs (`bool`, defaults to `True`):
                 Whether to cache block outputs to reuse as inputs for the succeeding block. It allows optimization of non-standard models
                 (e.g. ChatGLM) but can require more time.
+            inside_layer_modules (`List[List[str]]`, *optional*, defaults to `None`):
+                List of module names to quantize inside block_name_to_quantize. If not set, we will quantize all the linear layers.
```

Comment:
```suggestion
                List of module names to quantize inside block_name_to_quantize. This argument is useful to exclude certain linear modules from being quantized. If `None`, we will quantize all the linear layers. Example: `inside_layer_modules=[["self_attention.query_key_value"], ["mlp.dense_h_to_4h"]]`
```

Reply:
Why can't we just use `block_name_to_quantize`?

Reply:
Yeah this will be a lot better compared to using the same one as AutoGPTQ